.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     container:
-      image: registry.gitlab.com/finestructure/spi-base:0.8.0
+      image: registry.gitlab.com/finestructure/spi-base:0.9.2
```

Comment:
I wonder if this was the cause for the recent TSAN issues when I first tried to fix #1470: we were still using the 0.8.0 base image with the pre-release 5.7 nightly in our CI!